### ISSUE TITLE
couldn't I call SplashScreen show again?

### ISSUE THREAD

**Issue by L-Jovi** [crazycodeboy/react-native-splash-screen#39]
hi, thanks for your smart project,

I want to implement that show SplashScreen again when app switch to background, I try to call the method in `applicationDidEnterBackground` but failed.

what should I do?

thanks for your time.

regards.

**Comment by Jarred-Sumner** [comment]
experiencing this too! may submit PR later but we shall see

**Comment by parth19** [comment]
Exactly, I also tried finding quite a few ways to **show splash screen when appState is background**.
The closest I could come was using this react-native-splash-screen module, only to know that we **can not use the show() method to show the Splash screen again** :(

It is giving "**_reactNativeSplashScreen2.default.show is not a function**" error.

@Jarred-Sumner can you give me idea to solve this error and let me know if my required feature is possible or not with current build?

@L-Jovi Are you able to find anything else to solve this issue? How did you implement the "Show splash screen when app is in background" feature?

Thank you for your time. :)

**Comment by L-Jovi** [comment]
@parth19  no idea :(  eventually I cover the same image to the top of layer when I switch to background.

**Comment by ryanvanderpol** [comment]
I'd also like to have the ability to show the splash screen again from the RN side.  I use CodePush to do OTA updates that restarts the RN part of my app when restoring from the background when an update is applied, which I'd like to hide by showing the splash screen.

**Comment by andrejunges** [comment]
@ryanvanderpol have you been able to do a workaround this issue?
